cli/src/query.rs

```diff
@@ -0,0 +1,57 @@
+use nickel_lang_core::repl::query_print;
+
+use crate::{
+    cli::GlobalOptions,
+    error::{CliResult, ResultErrorExt},
+    eval::EvalCommand,
+};
+
+#[derive(clap::Parser, Debug)]
+pub struct QueryCommand {
+    pub path: Option<String>,
+
+    #[arg(long)]
+    pub doc: bool,
+
+    #[arg(long)]
+    pub contract: bool,
+
+    #[arg(long = "type")]
+    pub typ: bool,
+
+    #[arg(long)]
+    pub default: bool,
+
+    #[arg(long)]
+    pub value: bool,
+
+    #[command(flatten)]
+    pub evaluation: EvalCommand,
+}
+
+impl QueryCommand {
+    pub fn run(self, global: GlobalOptions) -> CliResult<()> {
+        let mut program = self.evaluation.prepare(&global)?;
+
+        program
+            .query(self.path)
+            .map(|term| {
+                // Print a default selection of attributes if no option is specified
+                let attrs =
+                    if !self.doc && !self.contract && !self.typ && !self.default && !self.value {
```

Comment:
I probably implemented that, but reading it again, might be clearer to move this condition into an associated method `no_attrs_specified(&self) -> bool` or whatever is a good name